tests/test_ansible.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import unittest
+import shlex
+from helper import shell_process
+import re
+
+"""Moneo ansible single node test"""
+
+
+class AnsibleTestCase(unittest.TestCase):
+    """Ansible unit test class"""
+    
+    def check_docker_status(self):
+        """Helper to check docker container status"""
+        cmd='sudo docker container ls'
+        args = shlex.split(cmd)
+        result = shell_process.shell_cmd(args)
+        if ('grafana' in result) or ( 'prometheus' in result ):
+            return True #deployed
+        else:
+            return False #not deployed
+            
+    def check_exporter_status(self):
+        """Helper to check exporter status"""
+        cmds=['ps -f -C  python2','ps -f -C  python3']
+        expected=['nvidia_exporter.py', 'net_exporter.py']
+        ret=False
+        for i in range(2):
```

Comment:
It may be worth changing this to `for i in range(len(cmds)):` if other exporters were to be added to this test.